pptx_generator/tpip_pptx/toc.py

```diff
@@ -42,6 +42,18 @@ def execute_table_drower(self,table, data,ids):
             cell_2.text = str(ids[row_id])
 
             row_index += 1
+            sub = pydash.get(row, "sub")
+            if sub != None:
```

Comment:
What happens if an empty sub list is provided? Please handle empty sub list checking also.